src/monodroid/monodroid.targets

```diff
@@ -22,22 +19,33 @@
         Overwrite="True"
     />
     <Exec
-        Command="$(AndroidToolchainDirectory)\ndk\ndk-build CONFIGURATION=$(Configuration) V=1"
+        Command="$(AndroidToolchainDirectory)\ndk\ndk-build CONFIGURATION=Debug V=1"
     />
     <Copy
         SourceFiles="@(AndroidSupportedTargetJitAbi->'obj\local\%(Identity)\libmonodroid.so')"
-        DestinationFiles="@(AndroidSupportedTargetJitAbi->'$(OutputPath)\%(Identity)\libmono-android.$(_Conf).d.so')"
+        DestinationFiles="@(AndroidSupportedTargetJitAbi->'$(OutputPath)\%(Identity)\libmono-android.debug.d.so')"
     />
     <Copy
         SourceFiles="@(AndroidSupportedTargetJitAbi->'libs\%(Identity)\libmonodroid.so')"
-        DestinationFiles="@(AndroidSupportedTargetJitAbi->'$(OutputPath)\%(Identity)\libmono-android.$(_Conf).so')"
+        DestinationFiles="@(AndroidSupportedTargetJitAbi->'$(OutputPath)\%(Identity)\libmono-android.debug.so')"
+    />
+    <Exec
+        Command="$(AndroidToolchainDirectory)\ndk\ndk-build CONFIGURATION=Release V=1"
```

Comment:
What this PR *wants* to do is generate both `libmono-android.debug.so` and `libmono-android.release.so`. As per above, this can't be achieved by using `CONFIGURATION`: it won't build.

Instead, I would suggest adding a separate make variable to `jni/Android.mk` which controls whether `-DRELEASE=1` is used, *separate from* `$(CONFIGURATION)`. This would allow you to do:

    $ ndk-build CONFIGURATION=Debug FASTDEV=1 V=1
    # copy to libmono-android.debug.so
    $ ndk-build clean
    $ ndk-build CONFIGURATION=Debug FASTDEV=0 V=1
    # copy to libmono-android.release.so